### PR TITLE
test uv for docs

### PR DESCRIPTION
**Pull request by Intron7**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -101,15 +101,6 @@ markers = [
...
 
-[tool.hatch.envs.hatch-test]
-features = ["test", "rapids12"]
-
-[tool.hatch.envs.docs]
-installer = "uv"
-features = ["doc"]
-scripts.build = "sphinx-build -M html docs docs/_build {args}"
-scripts.clean = "git clean -fXd docs"
-
```
Comment (flying-sheep) [inline pyproject.toml:112]: why?

File: .readthedocs.yml [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+  commands:
+    # Install and set up uv
+    - asdf plugin add uv
+    - asdf install uv latest
+    - asdf global uv latest
+
+    # Use uv to synchronize dependencies
+    - uv pip install --system .[doc]
+
+    # Build documentation using uv and sphinx
```
[the remark this answers is not in the document]
Comment (flying-sheep) [reply .readthedocs.yml:18]: I prefer it the way it is. `uv pip install --system` makes sense when running in a container.

I never understood why anyone would create a single venv in a container to run stuff from. Just use the global Python environment of the container!

File: pyproject.toml (hunk 1 above)
Comment (Intron7) [reply pyproject.toml:112]: I created a hatch.toml like scanpy and anndata has.

File: .readthedocs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Intron7) [reply .readthedocs.yml:18]: Than I'll leave it as is. I might adjust the comment

File: pyproject.toml (hunk 1 above)
Comment (flying-sheep) [reply pyproject.toml:112]: oof, seems like I’m more jetlagged than I thought. OK, sure haha

File: .readthedocs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ilan-gold) [reply .readthedocs.yml:18]: > Than I'll leave it as is. I might adjust the comment

Yea, my comment was more about the comment than actually doing it.  Cool.

[the remark this answers is not in the document]
Comment (flying-sheep) [reply .readthedocs.yml:18]: ah I missed that. yeah!

```suggestion
    # Build documentation using sphinx
```

**Pull request merged**